### DISCUSSION TITLE
Feeds from FlyterTalk website can't be used

### DISCUSSION THREAD

**Discussion by kpfleming** [FreshRSS/FreshRSS#6589] [Q&A] [answered]
I've just installed FreshRSS 1.24.1 and imported an OPML export from Liferea. Some of my feeds, from the FlyerTalk website, worked in Liferea but don't work in FreshRSS. When I click the 'Check the validity of the feed', I get a report indicating that is is valid, but with some 'recommendations'. An example is [here](https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fwww.flyertalk.com%2Fforum%2Fexternal.php%3Ftype%3DRSS2%26forumids%3D722).

How should I go about debugging this?

**Comment by math-GH** [comment]
Could you please check the logs (`/i/?a=logs`) too?

**Reply by kpfleming** [reply]
Well that's interesting... the logs show 403s for every attempt to access those URLs, but the same URL works in the W3C validator. Could this be a User-Agent being blocked or something similar?

**Reply by math-GH** [reply]
Have you tried different user agent strings?

**Reply by kpfleming** [reply]
I have not, I hadn't even considered that could be a problem until that test! I'll give something a try now.

**Reply by math-GH** [reply]
This could help too: https://github.com/FreshRSS/FreshRSS/discussions/6287

**Reply by kpfleming** [reply, marked as answer]
It's Cloudflare :( I'll install the Flaresolverr tool and extension, that should take care of the issue.
